frontend/src/lib/components/Sharing/SharingModal.tsx

```diff
@@ -174,9 +174,9 @@ export function SharingModalContent({
                                                         </div>
                                                     }
                                                     onChange={() =>
-                                                        guardAvailableFeature(AvailableFeature.WHITE_LABELLING, () =>
-                                                            onChange(!value)
```

Comment:
We should still call `onChange` here so it we ever save this to the API the form state is updated.

Reply:
`setEmbedConfigValue` is only calling `eventUsageLogic.actions.reportDashboardWhitelabelToggled(value)` so it is not updating the form state in kea.